tests/unit_tests/test_gameconnection.py

```diff
@@ -384,6 +384,7 @@ async def test_json_stats(game_connection: GameConnection, game_stats_service, p
 async def test_handle_action_EnforceRating(game: Game, game_connection: GameConnection):
     await game_connection.handle_action('EnforceRating', [])
     assert game.enforce_rating is True
+## MARPIE
```

Comment:
hmmmm

Reply:
Removed already, in the current state it is no longer there, my way of working might not be compatible. I did a reset HEAD~2 to remove all my commits, than made a fresh one. The github also says "Outdated".

Reply:
Yea I know, I was just looking at the commit history